app/controllers/admin/users_controller.rb

```diff
@@ -324,7 +324,12 @@ def silence
             silence_reason: full_reason,
             silenced_till: user.silenced_till,
             silenced_at: user.silenced_at,
-            silenced_by: BasicUserSerializer.new(current_user, root: false).as_json,
+            silenced_by:
+              BasicUserSerializer.new(
+                current_user,
+                root: false,
+                include_silence_reason: true,
```

Comment:
I do think this has to be tested at the controller level or an alternative is to just write a system test so that we cover the controller, serializer as well as the server side change in one go.